jsk_robot_common/jsk_owl_exporter/scripts/export_log.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Author: Yuki Furuta <furushchev@jsk.imi.i.u-tokyo.ac.jp>
+
+
+import argparse
+import os
+import sys
+from jsk_owl_exporter import *
+
+
+def list_tasks(db_addr, simple=False):
+    client = get_mongo_client(db_addr)
+    tasks = list_logged_tasks(client)
+    fmt = "{:<13}  ({:<6} docs, {:<10} - {:<10})"
+    if len(tasks) > 0:
+        for t in tasks:
+            if simple:
+                print t["task"]
+            else:
+                print fmt.format(t["task"], t["msg_count"], t["from"], t["till"])
+        return True
+    else: return False
+
+def info_task(db_addr, task_id):
+    client = get_mongo_client(db_addr)
+    info = get_logged_task_info(client, task_id)
+    stat = info.pop("Data")
+
+    # print result
+    fmt = "{:<13} | {:<30}"
+    for t, s in info.items():
+        print fmt.format(t, s)
+    print "Statistics"
+    statfmt = "  {:<50} | {:>6}"
+    print statfmt.format("Data Type", "Size")
+    for t, s in stat.items():
+        print statfmt.format(t, s)
+    return True
+
+def export_task(db_addr, task_id, output_dir):
+    client = get_mongo_client(db_addr)
+    agg = MongoAggregator(client, task_id)
+    node = agg.aggregate()
+    writer = OWLWriter(node)
+    save_path = os.path.join(output_dir, "log.owl")
+    writer.to_file(save_path)
+    print "saved to %s" % save_path
+    return True
+
+def graph_task(db_addr, task_id, output_dir, preview):
+    client = get_mongo_client(db_addr)
+    agg = MongoAggregator(client, task_id)
+    node = agg.aggregate()
+    writer = GraphWriter(node)
+    writer.parse()
+    save_path = os.path.join(output_dir, "%s.pdf" % task_id)
+    writer.save_pdf(save_path, preview)
+    print "saved to %s" % save_path
+    return True
+
+def exec_command(args):
+    if args.command == "list":
+        return list_tasks(args.db, args.simple)
+    elif args.command == "info":
+        return info_task(args.db, args.task)
+    elif args.command == "export":
+        return export_task(args.db, args.task, args.output)
+    elif args.command == "graph":
+        return graph_task(args.db, args.task, args.output, args.preview)
+
+
+if __name__ == '__main__':
+    p = argparse.ArgumentParser(description="OWL Exporter from mongo database")
+
+    p.add_argument("--db", default="mongodb://localhost:27017/jsk_robot_lifelog/pr1012",
```

Comment:
where is the `localhost` ? pr1012? or vm ? please add that information to README